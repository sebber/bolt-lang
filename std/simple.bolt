export fun test(): Integer {
    return 42
}