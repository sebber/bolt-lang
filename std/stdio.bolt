export fun print(message: String): Integer {
    return 0
}

export fun println(message: String): Integer {
    return 0
}