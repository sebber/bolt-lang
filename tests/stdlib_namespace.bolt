import simple from "bolt:simple"
import stdio from "bolt:stdio"

val result := simple.test()
stdio.print(result)
stdio.print("Standard library namespace test complete!")