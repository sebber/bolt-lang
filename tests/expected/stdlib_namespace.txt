42
Standard library namespace test complete!